server/service/team_policies.go

```diff
@@ -140,6 +180,111 @@ func (svc Service) DeleteTeamPolicies(ctx context.Context, teamID uint, ids []ui
 	if err := svc.authz.Authorize(ctx, &fleet.Policy{TeamID: ptr.Uint(teamID)}, fleet.ActionWrite); err != nil {
 		return nil, err
 	}
+	if len(ids) == 0 {
+		return nil, nil
+	}
+	ids, err := svc.ds.DeleteTeamPolicies(ctx, teamID, ids)
+	if err != nil {
+		return nil, err
+	}
+	if len(ids) == 1 {
+		err = svc.ds.NewActivity(
+			ctx,
+			authz.UserFromContext(ctx),
+			fleet.ActivityTypeDeletedPolicy,
```

Comment:
@gillespi314 @RachelElysia do we need to do anything in the frontend for the new activity items?

Reply:
@lucasmrod let's make sure we understand the above before merging this.

Reply:
Oooo good call @chiiph I can look into this this morning, probably just a new PR with some change in keys

Reply:
Indeed, see https://github.com/fleetdm/fleet/pull/2890#issuecomment-968965168.

Reply:
I'll revert the policy activities generation for a later iteration.